pom.xml

```diff
@@ -466,23 +472,6 @@
                             </execution>
                         </executions>
                     </plugin>
-                    <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-release-plugin</artifactId>
-                        <configuration>
-                            <preparationGoals>clean verify</preparationGoals>
-                            <prepareVerifyArgs>-Dmaven.test.skip=true</prepareVerifyArgs>
-                            <scmCommentPrefix>[ci skip]</scmCommentPrefix>
-                            <tagNameFormat>v@{project.version}</tagNameFormat>
-                        </configuration>
-                    </plugin>
-                    <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-deploy-plugin</artifactId>
```

Comment:
@hboutemy , isn't deploy plugin needed?
Why it should be removed?

Reply:
you declared it to configure skip parameter: configuring skip parameter is not needed, because nexus-staging-maven-plugin will do the job